kolibri/core/decorators.py

```diff
@@ -275,3 +276,19 @@ def wrapped_view(*args, **kwargs):
         return view_func(*args, **kwargs)
     wrapped_view.signin_redirect_exempt = True
     return wraps(view_func)(wrapped_view)
+
+
+def cache_no_user_data(view_func):
+    """
+    Set appropriate Vary on headers on a view that specify there is
+    no user specific data being rendered in the view.
+    """
+    def inner_func(*args, **kwargs):
+        request = args[0]
+        del request.session
+        response = view_func(*args, **kwargs)
+        headers = ['accept-encoding', 'accept', 'user-agent']
+        patch_response_headers(response, cache_timeout=300)
+        response['Vary'] = ', '.join(headers)
```

Comment:
could be simplified, unless the intent was to merge these headers with some existing headers?

```python
response['Vary'] = 'accept-encoding, accept, user-agent'
```